pkg/reconciler/contour/contour.go

```diff
@@ -223,10 +223,11 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, ing *v1alpha1.Ingress) r
 			return err
 		}
 		if diff, err := kmp.SafeDiff(update, matches[0]); err == nil {
-			logger.Debugf("Updated http proxy: %s", diff)
+			logger.Debug("Updated http proxy diff: ", diff)
```

Comment:
FYI 
L#226 and L#230 prints like logs this:
https://gist.githubusercontent.com/nak3/ebee40ab5c35a25abf4ea87982c0c3bd/raw/642b22546d5227ee2f527542cedd315d0f365d87/contour-httpproxy-example.log